setup.py

```diff
@@ -62,6 +62,8 @@
 install_requires = [
     "invenio-records-resources>=0.18.2,<0.19.0",
     "invenio-i18n>=1.3.1",
+    "lxml>=4.5.0",
```

Comment:
requires v4.5.0 or higher, otherwise it fails to install. see https://github.com/inveniosoftware/invenio-vocabularies/actions/runs/1687543630